### DISCUSSION TITLE
How to remove played audio files ?

### DISCUSSION THREAD

**Discussion by MartyLake** [PaulWoitaschek/Voice#1562] [Q&A] [answered]
Dear community,

I am using Voice as a fosss podcast player. I use yt-dlp to get the audio of podcasts I like from youtube, then sync them to a syncthing folder, they appear in Voice, I play them. Amazing.

But how can I remove all the played audio files ?

Best

**Comment by PaulWoitaschek** [comment, marked as answer]
You can long click on a book and then press on delete.

**Comment by MartyLake** [comment]
Thank you very much. Is it possible to select multiple books to delete them at the same time ?
Also, is it possible to scroll directly to "Completed" section, as I have 100+ podcasts in the queue and I have to manually scroll past them ?

**Reply by ghost** [reply]
AntennaPod is also a great app focused on podcasts instead of audiobooks. For your purposes I recommend AntennaPod.

**Reply by MartyLake** [reply]
Will try that, thank you very much for the suggestion!
